### PR TITLE
www.dtac.co.th

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119271

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-23 в 18 06 49

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -124,6 +124,8 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !
 serverguy.com##.elementor-popup-modal
 blog.paperspace.com###userpilotIframeContainer
+dtac.co.th#$##gridSystemModal { display: none !important; }
+dtac.co.th#$#.modal-open { overflow: hidden !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:128]: Is `hidden ` correct?

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -124,6 +124,8 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !
 serverguy.com##.elementor-popup-modal
 blog.paperspace.com###userpilotIframeContainer
+dtac.co.th#$##gridSystemModal { display: none !important; }
+dtac.co.th#$#.modal-open { overflow: auto !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:128]: It should be like below and I don't see the popup on desktop.
```
dtac.co.th#$##gridSystemModal { display: none !important; }
dtac.co.th#$#.modal-backdrop { display: none !important; }
dtac.co.th#$#body.modal-open { overflow: auto !important; padding-right: 0 !important; }
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — CHANGES_REQUESTED** [review]
Also remove this
<details><summary>Screenshot:</summary>

image

</details><br/>

File: MobileFilter/sections/specific_web.txt [hunk 3]
```diff
@@ -44,6 +44,9 @@ cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m
 !
 newsdig.tbs.co.jp##.ad_after_title
 xiaomiui.net##.bottom_fixad
+dtac.co.th#$##gridSystemModal { display: none !important; }
+dtac.co.th#$#.modal-backdrop { display: none !important; }
+dtac.co.th#$#body.modal-open { overflow: auto !important; padding-right: 0 !important; }
```
Comment (zloyden) [inline MobileFilter/sections/specific_web.txt:49]: Wrong file.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 3 above)
Comment (jellizaveta) [reply MobileFilter/sections/specific_web.txt:49]: In which file should I moved this rule?

**Review by zloyden — APPROVED** [review]

**Pull request merged**